runtime/j9vm/j7vmi.c

```diff
@@ -1525,6 +1525,12 @@ JVM_IHashCode(JNIEnv *env, jobject obj)
 jobject JNICALL
 JVM_InitProperties(JNIEnv* env, jobject properties)
 {
+	/* This JVM method is invoked by JCL native Java_java_lang_System_initProperties
+	 * only for initialization of platform encoding.
+	 * This is only required by Java 11 raw builds.
+	 * This method is not invoked by other Java levels.
```

Comment:
If this isn't invoked by other java levels, then why is most of the code only included in older levels?

Reply:
The code within this method can be removed, for example, `syspropsListRef` is defined within `J9JavaVM` but never initialized hence `assert(syspropsList != NULL)` will fire whenever `JVM_InitProperties` is called. I intended to leave the cleanup work to another PR as per this PR description (sorry for not making that obvious).